src/server/graph-generation/interaction/index.js

```diff
@@ -51,67 +56,76 @@ const getInteractionGraphFromPC = cache(rawGetInteractionGraphFromPC, pcCache);
  * @param {String[]} Array of query Ids
  * @return {json} JSON of graph
  */
-function parse(data, queryIds){
+function parse( data, queryIds ){
   let network = {
     edges:[],
     nodes:[],
   };
-  let nodeMap=new Map(); //keeps track of nodes that have already been added
-  if(data){
-    const dataSplit=data.split('\n\n');
-    const nodeMetadata= new Map(dataSplit[1].split('\n').slice(1).map(line =>line.split('\t')).map(line => [line[0], line.slice(1) ]));
-    dataSplit[0].split('\n').slice(1).forEach(line => {
+  let nodeMap = new Map(); //keeps track of nodes that have already been added
+  if( data ){
+
+    const nodeMetadata = null; //TODO: new service does not have node metadata
+    data.trim().split('\n').forEach( line => {
       const splitLine=line.split('\t');
-      const edgeMetadata = interactionMetadata(splitLine[6],splitLine[4]);
-      addInteraction([splitLine[0], splitLine[2]], splitLine[1], edgeMetadata, network, nodeMap, nodeMetadata, queryIds);
+      const nodes = [splitLine[0], splitLine[2]];
+      const edge = splitLine[1];
+      const mediatorIds = splitLine[4];
+
+      const edgeMetadata = interactionMetadata( mediatorIds );
```

Comment:
You're missing the second parameter, which should be `splitLine[4]` (see above).